src/mumble/NetworkConfig.ui

```diff
@@ -296,6 +296,29 @@ Prevents the client from downloading images embedded into chat messages with the
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="qgbPrivacy">
+     <property name="title">
+      <string>Privacy</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_4">
+      <item>
+       <widget class="QCheckBox" name="qcbHideOS">
+        <property name="toolTip">
+         <string>Prevent OS information being sent to the server</string>
+        </property>
+        <property name="whatsThis">
+         <string>&lt;b&gt;Don't send OS information to the server&lt;/b&gt;&lt;br/&gt;
+Prevents the client from sending potentially identifying information about the operating system to the server.</string>
+        </property>
+        <property name="text">
+         <string>Do not send OS information</string>
```

Comment:
Let's get this merged first, then we can work on it?

On a related note, I believe the change is starting to look ready... Could you squash the commits into one?